### PR TITLE
roadmap: update Tools

### PR DESCRIPTION
**Pull request by GregoryIan**
@iamxy @shenli PTAL

### CODE REVIEW

File: ROADMAP.md [hunk 1]
```diff
@@ -94,7 +94,8 @@ This document defines the roadmap for TiDB development.
 ## Tools:
 
 - [X] Tool for automating TiDB deployment
-- [X] High-Performance data import tool
-- [X] Backup and restore tool (incremental backup supported)
+- [X] High-Performance data import tool (lightning)
+- [X] Backup and restore tool (incremental backup supported by drainer, incrementtal restore supported by reparo)
+- [X] new TiDB-binlog with better architecture
```
Comment (iamxy) [inline ROADMAP.md]: s/better/improved

**Comment by morgo** [comment]
s/incrementtal/incremental/

**Comment by lilin90** [comment]
@GregoryIan Please address comments.

File: ROADMAP.md [hunk 2]
```diff
@@ -94,7 +94,8 @@ This document defines the roadmap for TiDB development.
 ## Tools:
 
 - [X] Tool for automating TiDB deployment
-- [X] High-Performance data import tool
-- [X] Backup and restore tool (incremental backup supported)
+- [X] High-Performance data import tool (lightning)
+- [X] Backup and restore tool (incremental backup supported by drainer, incremental restore supported by reparo)
+- [X] new TiDB-binlog with improved architecture
```
Comment (lilin90) [inline ROADMAP.md]:
```suggestion
- [X] New TiDB-binlog with improved architecture
```

**Pull request merged**